app/classifier/tasks/slider/index.jsx

```diff
@@ -0,0 +1,177 @@
+import { Markdown } from 'markdownz';
+import React from 'react';
+import GenericTask from '../generic';
+import SliderTaskEditor from './editor';
+
+const NOOP = Function.prototype;
+
+class Summary extends React.Component {
+  render() {
+    let answer = 'No answer';
+    if (this.props.annotation.value !== null) {
+      answer = this.props.annotation.value;
+    }
+    return (
+      <div>
+        <div className="question">
+          <Markdown tag="span" inline={true}>
+            {this.props.task.instruction}
+          </Markdown>
+        </div>
+        <div className="answer">
+          {answer}
+        </div>
+      </div>
+    );
+  }
+}
+Summary.displayName = 'SliderSummary';
```

Comment:
Renaming the component from `Summary` to `SliderSummary` will automatically set the display name.